lib/carmen/rails/action_view/form_helper.rb

```diff
@@ -207,13 +207,36 @@ def to_region_select_tag(parent_region, options = {}, html_options = {})
 
             value = options[:selected] ? options[:selected] : value(object)
             priority_regions = options[:priority] || []
-            opts = add_options(region_options_for_select(parent_region.subregions, value, 
-                                                        :priority => priority_regions), 
+            opts = add_options(region_options_for_select(parent_region.subregions, value,
+                                                         :priority => priority_regions),
                                options, value)
             select = content_tag("select", opts, html_options)
             if html_options["multiple"] && options.fetch(:include_hidden, true)
-              tag("input", :disabled => html_options["disabled"], :name => html_options["name"], 
-                           :type => "hidden", :value => "") + select
+              tag("input", :disabled => html_options["disabled"], :name => html_options["name"],
+                  :type => "hidden", :value => "") + select
+            else
+              select
+            end
+          end
+        end
+      end
+    end
+
+    if Rails::VERSION::MAJOR == 5
```

Comment:
this is there changes for rails 5 support, others changes are automatic code formate by ide

Reply:
I suggest you get rid of the automatic code format thing, because it is not functional. If there is any code to clean up you should do it at another time or at least in a separate commit